hana/packages.sls

```diff
@@ -14,6 +14,8 @@ patterns-sap-hana:
     - retry:
         attempts: 3
         interval: 15
+    # SAPHanaSR-ScaleOut conflicts with patterns-sap-hana and will be uninstalled (which will affect a running cluster)
+    - unless: rpm -q SAPHanaSR-ScaleOut
```

Comment:
I'm not sure about this change.
This looks a bug in our pattern or the package in any case, we shouldn't have this conflict

Reply:
After a discussion with the maintainer of the `patterns-sap-hana` pacakges, this seems to be an unreasonable conflict indeed at the moment.
Maybe a solution will come in the future but for now we have to do it this way.